packages/expo-clipboard/ios/ClipboardModule.swift

```diff
@@ -1,22 +1,86 @@
 // Copyright 2018-present 650 Industries. All rights reserved.
 
 import ExpoModulesCore
+import UIKit
 
 let onClipboardChanged = "onClipboardChanged"
 
 public class ClipboardModule: Module {
   public func definition() -> ModuleDefinition {
     name("ExpoClipboard")
 
+    // MARK: Strings
+
     function("getStringAsync") { () -> String in
       return UIPasteboard.general.string ?? ""
     }
 
-    function("setStringAsync") { (content: String?) in
+    function("setStringAsync") { (content: String?) -> Bool in
       UIPasteboard.general.string = content ?? ""
       return true
     }
 
+    function("hasStringAsync") { () -> Bool in
+      return UIPasteboard.general.hasStrings
```

Comment:
Here the `return` keyword is redundant. I'm not sure if this will look better without, but I think it is to consider this option too 🤔

Reply:
- I like the `return` more
- On the other hand, Kotlin requires either no return or qualified `return@function`. There I have the following "policy":
   - no return when something is obvious (like in this case)
   - `return@function` when it looks weird, like:
      ```kt
      function("doStuff") {
        // ...
        true // <--- IMO this looks weird
      }
      ```